src/CommonDBTM.php

```diff
@@ -6508,4 +6508,126 @@ public function isGlobal(): bool
 
         return $is_global;
     }
+
```

Comment:
good for the factorisation but not in commonDBTM please.
As said in GLPI mattermost channel today, a trait could be the way